clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -309,7 +310,9 @@
             FencedInstanceIdException::new),
     ELIGIBLE_LEADERS_NOT_AVAILABLE(83, "Eligible topic partition leaders are not available",
             EligibleLeadersNotAvailableException::new),
-    ELECTION_NOT_NEEDED(84, "Leader election not needed for topic partition", ElectionNotNeededException::new);
+    ELECTION_NOT_NEEDED(84, "Leader election not needed for topic partition", ElectionNotNeededException::new),
+    NO_REASSIGNMENT_IN_PROGRESS(85, "A partition reassignment is not in progress.",
```

Comment:
We can't have it be plural because this error would be returned for a single partition reassignment only.

I like the `No partition reassignment in progress` error message - I'll rename it to that